### PR TITLE
Change config & Dockerfile to use Dogecoin software

### PR DESCRIPTION
**Pull request by xanimo**
Fixes # .

Instead of building from source added extraction of dogecoind from dogecoin-1.14.3-x86_64-linux-gnu.tar.gz to app/rosetta-dogecoin.

### CODE REVIEW

**Review by patricklodder — CHANGES_REQUESTED** [review]
I would drop the automated text search & replace because a lot of things are unnecessary changes that we do not need to maintain. Especially the automated go package rename loses track of what can be using upstream code and what is truly custom. Would strongly recommend taking those out.

**Comment by xanimo** [comment]
Definitely will make those changes, should I close this and open a new PR or revert my local, fix the Dockerfile, commit and push?

**Comment by xanimo** [comment]
I'm not sure why but the new line at EOF of Dockerfile is not being pushed.  I reverted back to the last commit for the PR you approved, merged with upstream and then just added my changes to the Dockerfile.  Hopefully this one works.

File: Dockerfile [hunk 1]
```diff
@@ -81,12 +77,12 @@ RUN mkdir -p /app \
...
+COPY --from=bitcoind-builder /app/dogecoind /app/dogecoind
 
 # Copy binary from rosetta-builder
 COPY --from=rosetta-builder /app/* /app/
 
 # Set permissions for everything added to /app
 RUN chmod -R 755 /app/*
 
-CMD ["/app/rosetta-bitcoin"]
+CMD ["/app/rosetta-dogecoin"]
```
[the remark this answers is not in the document]
Comment (xanimo) [reply Dockerfile]: I commented below, the new line at EOF got automatically removed when I pushed, I can manually add a space if need be?

**Comment by xanimo** [comment]
Just realized I did not do any work within the assets folder to reflect dogecoin.

**Comment by xanimo** [comment]
`docker run -d --rm --ulimit "nofile=10000:10000" -v "$(pwd)/dogecoin-data:/data" -e "MODE=OFFLINE" -e "NETWORK=TESTNET" -e "PORT=8080" -p 8080:8080 rosetta-dogecoin:latest`
Screenshot (354)

Just to clarify for those wanting to replicate either clone my repo or copy my Docker, replace dogecoin-data with bitcoin-data above.  I wanted to ask if this is a good idea or is this unnecessary?

I would have to add dogecoin-data to .gitignore and .dockerignore

Outside of that I got it running kind of!

**Review by patricklodder — APPROVED** [review]
Works for me (offline) - thanks!

**Pull request merged**